docs/modules/configuration/pages/requisitions.adoc

```diff
@@ -0,0 +1,17 @@
+= Requisitions
+:imagesdir: ../assets/images
+
+In order to monitor the *Velocloud* Gateways, Edges and Links, these entities need to be imported into *OpenNMS*.
+You can achieve this by creating a requisition for a given connection alias.
+
+By specifying the requisition type `velocloud-customer` you can import all entities of a customer connection:
```

Comment:
This needs a line talking about the different access levels a connection can have and how they relate to choosen requisition type.

Reply:
The customer requisition can be used with a partner connection by specifying the enterprise id as additional parameter.

Reply:
done.